### PR TITLE
Let Slip the Dogs of War

### PR DESCRIPTION
**Pull request by CharlesWedge**
## The Machines Rise
With the recent sector wide hack rogue synthetics have become a problem in the sector. What's worse now corrupted fabricators are even building more combat designs! With an increasingly dangerous galaxy, it seems that mercenaries will not be the only threat brave explorers and security teams may face today. All security forces are advised to stay on the lookout for the latest threat to the galaxy, and those not equipped to take them on are advised, stay well out range!

## Why It's Good For The Game
As we want to move away for humanoid threats for simple mobs, I feel it necessary to shore up killer machines as more advanced type of enemy to take the place of humans in terms of  'dangerous opponents'. The current selection of machines mobs tend to be more niche in function (we can't exactly use the nanite horror as common enemies). Also there is a bigger maint drone now because the smaller ones weren't bad enough, least these ones are easier to hit.

## Changelog

:cl:
add: 4 New Hostile Drones, Three Dogs (including one sniper), and a Maint Drone.
add: New News article elaborating on recent events.
tweak: drones are now part of the same faction as hivebots. This means evil bots will now cooperate (Hivebots are being updated next).
/:cl:

### CODE REVIEW

File: code/modules/lore_codex/news_data/main.dm [hunk 1]
```diff
@@ -650,3 +650,25 @@
 	Nanotrasen for helping perserve the Scorian culture and expressed interest in further exploration fo Surt and its people. Though they, hestitated to use the word uplift they \
 	acknowledge that the Scori's integration with the galaxy is at this point inevitable even if they hoped it would be no time soon. At least the Scori then can rest easy perhaps \
 	never knowing how close they came to extinction yet again."
+
+/datum/lore/codex/page/hack
+	name = "Sector Spanning Cyber Attack Turns Synths Rogue! Hephaestus Combat Drones Designs Stolen! - 8/20/2567"
+	data = "Yesterday in the Frontier a massive cyber attack occured effecting an entire sector. Positronic intelligences, drones, ship AIs and all even MMI units were all said to \
+	be infected by  a malignant virus that was spread across the sector through subspace relays. Subverted silicons from the simplest drone to ship AIs were driven to stockpile materials \
```
Comment (BlueWildrose) [inline code/modules/lore_codex/news_data/main.dm:657]:
```suggestion
	be infected by a malignant virus that was spread across the sector through subspace relays. Subverted silicons from the simplest drone to ship AIs were driven to stockpile materials \
```

File: code/modules/lore_codex/news_data/main.dm [hunk 2]
```diff
@@ -650,3 +650,25 @@
...
+	to fuel corrupted drone fabricators which then began producing armies of corrupted drones. Infected units were also compelled to construct communication systems which transmitted \
+	data back towards what can almost be assumed to be the source of the virus. Casulties are thankfully limited however for Hephaestus Industries, who relies heavily on its drone \
+	employees,the loss of life and destruction of assests is thought to be immense. \
+	<br><br>\
+	A Hephaestus Industry vessel, HIV Tungsten Heart, has been reportedly been completely siezed by its subverted silicon crew, its organic crew thought to already be \
+	slaughtered. The now rogue vessel, was home to several drone fabrication facilities and to make matters worse, the databases of the fabricators allegedly contain blueprints \
+	for an as of yet unreleashed line of Hephaestus combat drones! These 'Warhound' series combat drones are armed with a variety od powerful mounted weapons meant to make short work \
```
Comment (BlueWildrose) [inline code/modules/lore_codex/news_data/main.dm:664]:
```suggestion
	for an as of yet unreleased line of Hephaestus combat drones! These 'Warhound' series combat drones are armed with a variety of powerful mounted weapons meant to make short work \
```

File: code/modules/mob/living/simple_mob/subtypes/mechanical/corrupt_maint_drone_vr.dm [hunk 3]
```diff
@@ -6,9 +6,9 @@
 
 /datum/category_item/catalogue/technology/drone/corrupt_maint_drone
 	name = "Drone - Corrupted Maintenance Drone"
-	desc = "This drone appears to be a station maintenance drone, produced by some sort of corrupt fab, \
-	which has caused it to become corrupt and attack anything nearby, except spiders and such, oddy. \
-	If one is found, a swarm of others are not too far away.\
+	desc = "This drone appears to be a station maintenance drone, produced by some sort of corrupt fab. \
+	Now this drone seems to target any organic life it comes across, working with others of its kind to overwhealm \
```
Comment (BlueWildrose) [inline code/modules/mob/living/simple_mob/subtypes/mechanical/corrupt_maint_drone_vr.dm:10]:
```suggestion
	Now this drone seems to target any organic life it comes across, working with others of its kind to overwhelm \
```

File: code/modules/mob/living/simple_mob/subtypes/mechanical/corrupt_maint_drone_vr.dm [hunk 4]
```diff
@@ -6,9 +6,9 @@
...
+	Larger foes, if one is found, a swarm of others are not too far away.\
```
Comment (BlueWildrose) [inline code/modules/mob/living/simple_mob/subtypes/mechanical/corrupt_maint_drone_vr.dm:11]:
```suggestion
	larger foes. If one is found, a swarm of others are not too far away.\
```
If this is part of the same sentence, remove the capitalization.

**Review by BlueWildrose — CHANGES_REQUESTED** [review]
Some proofreading for you.

**Review by Captain277 — APPROVED** [review]

**Pull request merged**